src/main/kotlin/io/titandata/titan/utils/CommandExecutor.kt

```diff
@@ -27,7 +28,9 @@ class CommandExecutor(val timeout: Long = 10) {
         builder.command(args)
         val process = builder.start()
         try {
-            process.waitFor(timeout, TimeUnit.MINUTES)
+            if (SystemUtils.IS_OS_MAC || SystemUtils.IS_OS_LINUX) {
+                process.waitFor(timeout, TimeUnit.MINUTES)
+            }
```

Comment:
Did you ever figure out what was causing this? I'd add a detailed comment explaining why this is here as it won't be obvious from the context or PR history

Reply:
I meant to put a comment for this! Thanks.